system/modules/core/modules/ModuleBooknav.php

```diff
@@ -154,6 +155,7 @@ protected function getBookPages($intParentId, $groups, $time)
 		{
 			while ($objPages->next())
 			{
+				/** @var \PageModel $objPages */
```

Comment:
if `\PageModel::findPublishedSubpagesWithoutGuestsByPid` would `@return \Model\Collection|static[]` and we'd use `foreach` that would not be necessary.